src/main/java/dev/blaauwendraad/masker/json/KeyMatcher.java

```diff
@@ -173,6 +173,29 @@ public KeyMaskingConfig getMaskConfigIfMatched(byte[] bytes, int keyOffset, int
         }
     }
 
+    /**
+     * An overload of {@code dev.blaauwendraad.masker.json.KeyMatcher#getMaskConfigIfMatched(byte[], int, int, java.util.Iterator)} that does only jsonpath lookup.
+     */
```

Comment:
> that does only jsonpath lookup.

=> "specifically for JsonPath look-ups?

Reply:
That is a better wording, I will use it.